src/stream/src/task/barrier_manager.rs

```diff
@@ -1033,6 +892,13 @@ impl LocalBarrierManager {
         });
         rx
     }
+
+    pub fn register_sender(&self, actor_id: ActorId, tx: UnboundedSender<Barrier>) {
+        self.send_event(LocalBarrierEvent::RegisterBarrierSender {
```

Comment:
Can we also update the interface to return an `rx`? So that the caller doesn't need to create a channel pair itself.